test/data/image-trivyignores.test

```diff
@@ -75,15 +75,12 @@ Total: 19 (CRITICAL: 19)
 
 rust-app/Cargo.lock (cargo)
 ===========================
-Total: 2 (CRITICAL: 2)
+Total: 1 (CRITICAL: 1)
 
 ┌──────────┬────────────────┬──────────┬───────────────────┬───────────────┬─────────────────────────────────────────────────────────────┐
 │ Library  │ Vulnerability  │ Severity │ Installed Version │ Fixed Version │                            Title                            │
 ├──────────┼────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────────┤
-│ openssl  │ CVE-2018-20997 │ CRITICAL │ 0.8.3             │ 0.10.9        │ Use after free in openssl                                   │
-│          │                │          │                   │               │ https://avd.aquasec.com/nvd/cve-2018-20997                  │
-├──────────┼────────────────┤          ├───────────────────┼───────────────┼─────────────────────────────────────────────────────────────┤
-│ smallvec │ CVE-2021-25900 │          │ 0.6.9             │ 1.6.1, 0.6.14 │ An issue was discovered in the smallvec crate before 0.6.14 │
+│ smallvec │ CVE-2021-25900 │ CRITICAL │ 0.6.9             │ 1.6.1, 0.6.14 │ An issue was discovered in the smallvec crate before 0.6.14 │
```

Comment:
It is weird. The number of vulnerabilities should be increased. This bump made less.

Reply:
Seems like CVE-2018-20997 should not be detected.
https://github.com/advisories/GHSA-xjxc-vfw2-cg96

Reply:
It was just a mistake of GitHub. Trivy is not wrong.